plugins/modules/vmware_vm_info.py

```diff
@@ -326,6 +328,17 @@ def get_virtual_machines(self):
             if self.module.params.get('show_tag'):
                 vm_tags = self.get_tag_info(vm)
 
+            storage_allocated = 0
+            for device in vm.config.hardware.device:
+                if isinstance(device, vim.vm.device.VirtualDisk):
+                    storage_allocated += device.capacityInBytes
+
+            allocated = {
+                "storage": storage_allocated,
+                "cpu": vm.config.hardware.numCPU,
+                "memory": vm.config.hardware.memoryMB
+                }
```

Comment:
ERROR: plugins/modules/vmware_vm_info.py:340:17: E123: closing bracket does not match indentation of opening bracket's line